packages/framework/attributor/src/encoders.ts

```diff
@@ -108,6 +166,46 @@ export class AttributorSerializer implements IAttributorSerializer {
 			const user = JSON.parse(interner.getString(ref)) as IUser;
 			entries[i] = [key, { user, timestamp }];
 		}
+
+		if (customAttributionInfo !== undefined) {
+			const {
+				ids,
+				timestamps: encodedTimestamps2,
+				attributionRefs: attributionRefs2,
+				attributionPropsRefs,
+				attributionValuesRefs,
+			} = customAttributionInfo;
+			const timestamps2 = this.timestampEncoder.decode(encodedTimestamps2);
+			assert(
+				ids.length === timestamps2.length &&
```

Comment:
maybe cache ids.length into local variable and use it below? Complier will use single letter variable for it - a bit more compact.